scripts/skills/actives/rf_net_pull_skill.nut

```diff
@@ -41,7 +41,7 @@ this.rf_net_pull_skill <- ::inherit("scripts/skills/skill", {
 			id = 10,
 			type = "text",
 			icon = "ui/icons/vision.png",
-			text = "Has a range of " + ::MSU.Text.colorPositive("2") + " tiles"
+			text = "Has a range of " + ::MSU.Text.colorPositive("2") + " tiles and max height difference of 1"
```

Comment:
I'm skeptical about adding the height difference. It isn't vanilla convention and isn't mentioned for any other 2 tile skill. In most cases this is automatically implied and learned by BB players that 2 tile melee skills have a height difference of 1. I'd say don't add this.